### PR TITLE
login and user page

### PR DESCRIPTION
**Pull request by pradeep-gururani**

### CODE REVIEW

File: src/components/ListPoll.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+class ListPoll extends Component {
+    render() { 
+        console.log('list',this.props.state.listPoll.data.data);
+        
+        return (
+            <div className = "col-sm-9">
+                <div className="accordion">                                           
+                            {this.props.state.listPoll.data.data.map(dat=>(
+                            <div className="card" key ={dat}> 
+                                <div class="card-header" id={dat}>
```
Comment (arun-kumar-codes) [inline src/components/ListPoll.js]: why class not className

File: src/redux/login/action.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+import { put } from 'redux-saga/effects'
+import * as actions from '../actions';
+import jwt_decode from  'jwt-decode';
+export function* loginRequest(action){
+    let error = null;
+    let url = `https://secure-refuge-14993.herokuapp.com/login?username=${action.payload.user}&password=${action.payload.passwd}`;
```
Comment (arun-kumar-codes) [inline src/redux/login/action.js]: move this to config file

Comment (pradeep-gururani) [reply src/redux/login/action.js]: ok sir

File: src/components/ListPoll.js (hunk 1 above)
Comment (pradeep-gururani) [reply src/components/ListPoll.js]: I missed updating it, I will fix  it.

**Pull request closed without merging**